File: atinject-core/src/main/java/org/atinject/core/latency/dto/PingRequest.java
package org.atinject.core.latency.dto;

import org.atinject.core.websocket.dto.WebSocketRequest;

public class PingRequest extends WebSocketRequest {

    private static final long serialVersionUID = 1L;

    private long time;

    public long getTime() {
        return time;
    }

    public PingRequest setTime(long time) {
        this.time = time;
        return this;
    }
    
}

File: atinject-integration/src/main/resources/webapp/scripts/atinject/latency/PingRequest.js
define(["atinject/WebSocketRequest"],
function(WebSocketRequest){
    function PingRequest() {
        this._class = "org.atinject.core.latency.dto.PingRequest"
    };

    PingRequest.prototype = new WebSocketRequest();

        PingRequest.prototype.getTime = function() {
        return this.time;
    };

    PingRequest.prototype.setTime = function(time) {
        this.time = time;
        return this;
    };


    
    return PingRequest;
});
